Add Diggie to the partners page

lopburny opened the pull request:
### Description
Hi folks, Thank you for all the work. 
Adding Diggie logo and link to the partners page. 

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

stockholmux commented:
Thanks for the PR @lopburny! 


I see the logo and the link go to Diggie in this PR.Usually the partners pages link to your company page and uses your companies logo. 

If you want to feature your project, you can add it to the [Community Projects page](https://opensearch.org/community_projects).

What do you think?

lopburny commented:
Updated our company logo and link, also moved Diggie to the community projects page! 
One thing, we're currently working on the license text, is it OK to list the projects with a blank license link first, and update it in a few weeks?

stockholmux commented on _community_projects/diggie.md:
```diff
@@ -0,0 +1,12 @@
+link: https://github.com/DiggieApp/Diggie
+download_link: https://github.com/DiggieApp/Diggie/releases/tag/0.1.0
+license: Diggie EULA (developer preview)
+license_link: "#" # in preparation
```
I'd rather this lead to a 404 than to `#`. If you know the URL where your license will be already, then just put it in now and we don't have to change it later.

lopburny replied:
@stockholmux I get it 🙌  Added license_link and we'll make the link available soon!

stockholmux reviewed (approved):
LGTM!
